app/templates/views/your-services.html

```diff
@@ -36,19 +36,20 @@ <h2>
 
 {% macro no_service(
   heading,
-  no_live_service_text
+  heading_classes,
+  no_services_text
 ) %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-one-quarter">
-      <h2>
+      <h2 class="{{heading_classes}}">
```

Comment:
```suggestion
      <h2 {% if heading_classes %}class="{{heading_classes}}"{% endif %}>
```

Not a blocker but there is a potential chance of no classes being passed in so we may want to not include a class attribute if there isn't.

Reply:
thanks @DilwoarH i've merged you suggested and waiting for tests to pass